experimental/report/diff.go

```diff
@@ -93,11 +93,11 @@ func unifiedDiff(span Span, edits []Edit) (Span, []hunk) {
 			strings.Contains(src[edit.End:next.Start], "\n")
 	})
 
-	var out []hunk
+	var out []hunk //nolint:prealloc // False positive.
```

Comment:
No heuristic?

Reply:
Calculating a heuristic will be O(n); not preallocating has O(log n) allocation cost. I don't think it's worth bothering.